### PR TITLE
Test with PHPUnit 8

### PR DESCRIPTION
**Pull request by greg-1-anderson**
### Overview
This pull request:

- [ ] Fixes a bug
- [ ] Adds a feature
- [ ] Breaks backwards compatibility
- [x] Has tests that cover changes

### Summary
Modernize and use phpunit 7 and 8

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          extensions: ${{ env.PHP_EXTENSIONS }}
+          ini-values: ${{ matrix.php-ini-values }}
+
+      - name: Determine composer cache directory on Linux
+        if: matrix.os == 'ubuntu-latest'
+        run: echo "COMPOSER_CACHE_DIR=$(composer config cache-dir)" >> $GITHUB_ENV
+
+      - name: Determine composer cache directory on Windows
+        if: matrix.os == 'windows-latest'
+        run: ECHO "COMPOSER_CACHE_DIR=~\AppData\Local\Composer" >> $GITHUB_ENV
```
Comment (marktheunissen) [inline .github/workflows/ci.yml:113]:
```suggestion
        run: $ENV:GITHUB_ENV += "COMPOSER_CACHE_DIR=~\AppData\Local\Composer"
```

Comment (marktheunissen) [inline .github/workflows/ci.yml]: A second option which might work, assuming the `if` statement holds, is to just have this:

```
- name: Determine composer cache directory on Windows
         if: matrix.os == 'windows-latest'
         env:
           COMPOSER_CACHE_DIR: ~\AppData\Local\Composer
```

Comment (marktheunissen) [inline .github/workflows/ci.yml:113]:
```suggestion
        run: ECHO "COMPOSER_CACHE_DIR=~\AppData\Local\Composer" >> $GITHUB_ENV
        run: Add-Content -Path $ENV:GITHUB_ENV -Value "COMPOSER_CACHE_DIR=~\AppData\Local\Composer"
```

Comment (marktheunissen) [inline .github/workflows/ci.yml]:
```suggestion
        run: Add-Content -Path $ENV:GITHUB_ENV -Value "COMPOSER_CACHE_DIR=~\AppData\Local\Composer"
```

**Review by weitzman — APPROVED** [review]
LGTM. Great work.

**Pull request merged**